.storybook/preview.js

```diff
@@ -1,4 +1,12 @@
 import { addDecorator } from '@storybook/react';
+import { addParameters } from '@storybook/client-api';
 import { withPerformance } from '../src';
+import { defaultAllowedGroups } from '../src/tasks/preset';
+
+addParameters({
+  performance: {
+    allowedGroups: defaultAllowedGroups,
```

Comment:
I don't think we should have to provide this

Reply:
This example was more for example purposes. It works perfectly fine without specifying `allowedGroups` as it defaults to `defaultAllowedGroups`. I've removed this in the latest commit to reduce confusion for future readers.